src/main/java/pl/futurecollars/invoicing/config/Config.java

```diff
@@ -0,0 +1,8 @@
+package pl.futurecollars.invoicing.config;
+
+public class Config {
+
+  public static final String DATABASE_LOCATION = "db/invoices.txt";
```

Comment:
Jeśli dane zapisane w pliku są w formacie json, to dajmy mu odpowiednie rozszerzenie (json).